package.json

```diff
@@ -2,6 +2,7 @@
   "name": "livetl.app",
   "version": "0.0.1",
   "private": true,
+  "prepare": "svelte-kit sync",
```

Comment:
Does this not need to be in `scripts` to actually work?

Reply:
Possibly, well we can fix that later ww

Reply:
Did it in https://github.com/LiveTL/website-new/commit/61231a19e85f3e3ad7203c3985c7884217933928